GoogleChromeLabs/ps-analysis-tool
# Fix: Revert removal of use arguments and update CLI build commands

## Description
This PR aims to revert the removal of the default optional argument from the CLI, which was published in 0.9 version.

## Relevant Technical Choices



## Testing Instructions
- Clone this branch.
- In the terminal run `npm run cli https://bbc.com`.
- The report should be generated after the CLI is done analysing the site.

## Additional Information:



## Screenshot/Screencast



---

## Checklist
